presto-manager-common/src/main/java/com/teradata/prestomanager/common/ApiRequester.java

```diff
@@ -32,7 +32,7 @@
 
 public class ApiRequester
 {
-    private final UriBuilder uriBuilder;
+    private final String uriTemplate;
```

Comment:
I'm storing the URI template now instead of the URI. This allows the `ApiRequester` to safely be used by different threads at the same time.